src/index.ts

```diff
@@ -26,4 +27,24 @@ app.use((_, res) => {
   rpcError(res, 404, {}, '');
 });
 
-app.listen(PORT, () => console.log(`Listening at http://localhost:${PORT}`));
```

Comment:
we are using this on all other servers i think, for example snapshot-hub, maybe not related 🤔

Reply:
what do you mean ?